### PR TITLE
Build script changes to skip prompt and set up api keys

### PR DESCRIPTION
**Pull request by chung-lai**
* Added prompt options to get all the interactive prompt and the script will skip most prompts by default (meaning less "Enter" key press :D )
* Added API keys support, will update build document for that after test run

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### Manual test cases
If manually tested, describe the different scenarios you followed to ensure
that your changes work.  Make sure that you've tested as many pathways as
possible -- not just the [happy path](https://en.wikipedia.org/wiki/Happy_path).

### Screenshots and screen recordings
Images or videos showing the effect of your PR.  Examples include:
* For new Views, add images of `_Previews` you have added for both light
  and dark mode
* Before and after screenshots showing the effect of your styling changes
* Videos showing you performing the flow that you have changed

## Issues addressed
* Link(s) to Github issue(s) (if applicable)

### CODE REVIEW

File: Scripts/build-release.sh [hunk 1]
```diff
@@ -2,27 +2,35 @@
...
-while getopts "ebv" option; do
+while getopts "ebp" option; do
   case $option in
     e) # export
        EXPORT_BUILD=true
        ;;
     b) # create branch
        CREATE_BRANCH=true
        ;;
+    p) # prompt on every build step
```
Comment (darinwf) [inline Scripts/build-release.sh:20]: nit: maybe go with "s" here for "skip" might be easier to remember. "p" sounds more like an option to ask for prompts :)

**Review by darinwf — APPROVED** [review]

File: Scripts/build-release.sh (hunk 1 above)
Comment (chung-lai) [reply Scripts/build-release.sh:20]: That's meant to be the case. :) You specify `-p` to get all the prompts, it skips all the prompts by default

Comment (chung-lai) [reply Scripts/build-release.sh:20]: Oh my description in the PR is wrong :P

Comment (darinwf) [reply Scripts/build-release.sh:20]: Oh lol

**Pull request merged**